src/app/directives/is-ha/is-ha.directive.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  Directive,
+  TemplateRef,
+  ViewContainerRef,
+  OnInit,
+  input,
+} from '@angular/core';
+import { UntilDestroy, untilDestroyed } from '@ngneat/until-destroy';
+import { Store } from '@ngrx/store';
+import { AppState } from 'app/store';
+import { selectIsHaLicensed } from 'app/store/ha-info/ha-info.selectors';
+
+@UntilDestroy()
+@Directive({
+  selector: '[isHa]',
+})
+export class IsHaDirective implements OnInit {
```

Comment:
Tests please.

Reply:
ah, sorry, missed that one, added tests for the components only 🙉